README.md

```diff
@@ -204,6 +204,20 @@ The `group` syntax is also possible in combination with `groupedbar`:
 ```julia
 groupedbar([1, 2, 1, 2, 1, 2], rand(6), group = [1, 1, 2, 2, 3, 3])
 ```
+A more complex example is also possible:
```

Comment:
Can you delete this and the example before? And consider whether one of the two previous graphs could be taken out as well? I don't think we need more than two plots for groupedbar, and they should ideally be combined to a two-panel plot (perhaps reusing the data, but letting e.g. one represent the column-matrix attributes and the minimum input, the other the grouped method).

Reply:
Sorry, I cannot understand the example you are giving. Can you please elaborate further?

Either way I already deleted this line and the example before.